apps/docs/content/components/Heading/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Heading
+description: Use the Heading component to render title or subtitle text
+---
+
+import CustomVideoPlayer from '../../../src/components/custom-video-player'
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+![An image displaying each header from h1 to h6.](https://github.com/primer/brand/assets/912236/daf645d9-2ce5-4aba-9bc0-7c95b401c6c8)
+
+## Usage
+
+Headings are used to structure the content on the page, with each heading representing a different level of importance. It is recommended to use headings in a logical manner to create a clear hierarchy and improve accessibility.
+
+- Heading should be used to communicate page organization and hierarchy.
+- Use heading as the title that introduces a page, a section or an article.
+- The heading component will render an html h2 tag without any default styling.
+
+## Options
+
+### Scale and size
+
+The Heading scale ranges from 1-6 and maps to the HTML heading elements h1-h6 respectively. The size is determined by the type of HTML element specified, although you can override this when the scenario calls for it. E,g. use a bigger heading size to highlight an article that might be rellevant to the user.
+
+### Weight, stretch and letter spacing
+
+The Heading component also supports the following options: `weight`, `stretch` and `letterSpacing`. Each of these options are determined by the heading scale and size, but can be overridden when the scenario calls for it.
+
+### Responsive
+
+The Heading component is responsive by default and the size will scale down on smaller screens. Weight, stretch and letter spacing will remain the same across all screen sizes but can be overridden for each breakpoint.
+
+<div>
+  <CustomVideoPlayer
+    width="100%"
+    src="https://github.com/primer/brand/assets/912236/d0e7ef2b-d52f-413e-8580-579a9129696c"
+  />
+</div>
+
+## Accessibility
+
+While sighted users rely on visual cues such as font size changes to determine what the heading is, assistive technology users rely on programatic cues that can be read out. When text on a page is visually implied to be a heading, ensure that it is coded as a heading. Additionally, visually implied heading level and coded heading level should be consistent. See [WCAG success criteria: 1.3.1: Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html).
+
+Headings allow assistive technology users to quickly navigate around a page. Navigation to text that is not meant to be a heading can be a confusing experience. [Learn more about best heading practices (WAI Headings)](https://www.w3.org/WAI/tutorials/page-structure/headings/).
```

Comment:
Should this be:

```suggestion
Headings allow assistive technology users to quickly navigate around a page. Navigation as text that is not meant to be a heading can be a confusing experience. [Learn more about best heading practices (WAI Headings)](https://www.w3.org/WAI/tutorials/page-structure/headings/).
```

Reply:
I think this is a good opportunity for a do/don't also, as this is really easy to get wrong.